frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/scheduler/KafkaService.java

```diff
@@ -27,14 +29,18 @@ protected void startApiServer(DefaultScheduler defaultScheduler,
                                   int apiPort,
                                   Collection<Object> additionalResources) {
         final ServiceSpec serviceSpec = super.getServiceSpec();
+
```

Comment:
Line break probably not necessary, seems reasonable to group all these declarations in a single paragraph.